substrate/client/consensus/beefy/src/keystore.rs

```diff
@@ -111,7 +111,15 @@ impl<AuthorityId: AuthorityIdBound> BeefyKeystore<AuthorityId> {
 				sig_ref.to_vec()
 			},
 
-			_ => Err(error::Error::Keystore("key type is not supported by BEEFY Keystore".into()))?,
```

Comment:
I am not adding a case for `bls381::CRYPTO_ID`, due it does not need additional hashing as I understand, please correct me if I am wrong :)